db/migrate/20160701193035_add_more_urls_to_external_activity.rb

```diff
@@ -0,0 +1,6 @@
+class AddMoreUrlsToExternalActivity < ActiveRecord::Migration
+  def change
+    add_column :external_activities, :author_url, :string
+    add_column :external_activities, :print_url, :string
```

Comment:
A couple of times now I've had to change urls from strings to text because string isn't long enough. So I'd recommend start these urls off as text.